docs/content/installation/using-gcp-marketplace-package.md

```diff
@@ -0,0 +1,138 @@
+---
+title: Using GCP Marketplace Ingress Controller
```

Comment:
should the page title be using-gcp-marketplace-package OR using-gcp-marketplace-ingress-controller-image, trying to keep inline with AWS one

Reply:
There is a pointer in the setup for the GCP listing that needs to be updated.
Be sure to poke me with the publishing URL once we settle.

Reply:
@brianehlert we're gonna use the one that is already in the listing, for now, to not delay the publishing even further. We can always have a redirect when we settle on a link and then update the listing after.